ts/DataGrid/DataGrid.ts

```diff
@@ -130,6 +179,7 @@ class DataGrid {
     }
 
 
+    /** @internal */
```

Comment:
Please add a description and check other places where lint complains about missing one.